whois-commons/src/main/resources/applicationContext-commons.xml

```diff
@@ -21,7 +21,7 @@
     <context:component-scan base-package="net.ripe.db.whois.common"/>
 
     <context:annotation-config/>
-    <cache:annotation-driven mode="aspectj"/>
```

Comment:
Do we not still need this?

Reply:
I wanted it to be as it was before spring upgrade.

Reply:
I think it's safer to be explicit, we weren't doing that before, which means we may have mixed AspectJ weaving and proxying. We should specify aspectj mode everywhere.

Reply:
Have a read of the issue https://github.com/spring-projects/spring-framework/issues/32882

"I still recommend revising your application setup towards exclusive weaving usage (no proxy setup at all)."

I think we're in the same situation.

Reply:
Agreed lets downgrade completely and add mode later when we re-upgrade

Reply:
Yes,  lets do it as part of spring upgrade.Thanks